client/src/components/ui/itemrow/SendRow.svelte

```diff
@@ -12,9 +18,22 @@
     export let doc: Document['id'];
     export let category: Category['name'];
     export let title: Document['title'];
-    export let target: Snapshot['target'];
+    export let target: Office['id'];
+    export let creation: Snapshot['creation'];
+    let targetName: Office['name'];
```

Comment:
See the [Svelte docs](https://svelte.dev/docs#component-format-script-3-$-marks-a-statement-as-reactive).
```suggestion
```